mava/systems/tf/mappo/training.py

```diff
@@ -48,32 +49,29 @@ class MAPPOTrainer(mava.Trainer):
     def __init__(
         self,
         agents: List[Any],
-        agent_types: List[str],
         observation_networks: Dict[str, snt.Module],
         policy_networks: Dict[str, snt.Module],
         critic_networks: Dict[str, snt.Module],
         dataset: tf.data.Dataset,
         policy_optimizer: Union[snt.Optimizer, Dict[str, snt.Optimizer]],
         critic_optimizer: Union[snt.Optimizer, Dict[str, snt.Optimizer]],
+        variable_client: VariableClient,
+        counts: Dict[str, Any],
         agent_net_keys: Dict[str, str],
-        checkpoint_minute_interval: int,
-        discount: float = 0.99,
-        lambda_gae: float = 1.0,
-        entropy_cost: float = 0.0,
-        baseline_cost: float = 1.0,
+        discount: float = 0.999,
+        lambda_gae: float = 1.0,  # Question (dries): What is this used for?
```

Comment:
Can we set the default to 0.95. This should be used in the advantage func calculation.

Reply:
Yes, this is used in the Generalized Return Estimation. But I think we are doing a double discount in the `trfl.td_lambda` function if we set `lambda_gae` not equal to 1.0. We should probably investigate this again.

Reply:
I changed it to 0.95. Although I am still not sure why it should just not always equal 1.0.